grails-app/controllers/au/org/ala/spatial/portal/PortalController.groovy

```diff
@@ -376,28 +376,67 @@ class PortalController {
 
             def url = grailsApplication.config.lists.url
 
-            def header = [:]
-            if (Holders.config.security.oidc.enabled) {
-                header.put("userId", userId)
-                header.put("X-ALA-userId", userId)
-                header.put("apiKey", grailsApplication.config.api_key)
-                //header.put('Cookie', 'ALA-Auth=' + URLEncoder.encode(authService.email, 'UTF-8'))
+            def r = webService.post("${url}/ws/speciesList/", json)
```

Comment:
Are we going to to abandon HubWebserive in future, but use Webservice provided by ala-ws-plugin for Token support?

Reply:
Yes